Agents/HeatEmissionAgent/Dockerfile

```diff
@@ -21,11 +21,11 @@ RUN --mount=type=cache,target=/root/.m2/repository mvn package
 
 # Second stage: copy the downloaded dependency into a new image and build into an app
 #==================================================================================================
-FROM tomcat:8.5 as agent
+FROM tomcat:9.0 as agent
 
 WORKDIR /app
 
-COPY --from=builder /root/HeatEmissionAgent/output/heatemissionagent##1.0.0.war $CATALINA_HOME/webapps/
+COPY --from=builder /root/HeatEmissionAgent/output/heat-agent##2.0.0.war $CATALINA_HOME/webapps/
```

Comment:
Be consistent with the naming of the agent. Either "HeatEmissions Agent" or "Heat Agent". The former is more descriptive in my opinion

Reply:
The 'heatemissionagent' label is now used in all files.